Dockerfile

```diff
@@ -1,37 +1,72 @@
-FROM node:22-alpine as builder
+# Use the official Node.js 22 Image (Alpine version) as the base
+FROM oven/bun:1-alpine AS base
 
-ARG TURBO_VERCEL_TEAM_NAMESPACE
-ENV TURBO_TEAM=${TURBO_VERCEL_TEAM_NAMESPACE}
+# Create a layer for dependency installation
+FROM base as dependencies
 
-ARG TURBO_VERCEL_REMOTE_CACHE_TOKEN
-ENV TURBO_TOKEN=${TURBO_VERCEL_REMOTE_CACHE_TOKEN}
+# Create our working directory
+RUN mkdir -p /home/app/project-arcturus
 
-ENV TURBO_REMOTE_CACHE=true
+# Move over to our working directory
+WORKDIR /home/app/project-arcturus
 
-RUN echo "${TURBO_REMOTE_CACHE}"
-RUN echo "${TURBO_TEAM}"
-RUN echo "${TURBO_TOKEN}"
-
-# Set PNPM Env Variables, Necessary for PNPM setup in Docker Env
-ENV PNPM_HOME="/pnpm"
-ENV PATH="$PNPM_HOME:$PATH"
-ENV RUNTIME_STAGE="production"
+# Copy our turborepo workspace root package.json
+COPY ./package.json ./package.json
 
-RUN corepack enable
+# Copy our turborepo workspace root bun.lockb
+COPY ./bun.lockb ./bun.lockb
 
-RUN mkdir -p /home/app/turbo
+# Copy our turborepo workspace root turbo.json
+COPY ./turbo.json ./turbo.json
 
-WORKDIR /home/app/turbo
+# Copy our website
+COPY ./apps/web ./apps/web
 
-COPY ./package.json ./package.json
-COPY ./pnpm-lock.yaml ./pnpm-lock.yaml
-COPY ./pnpm-workspace.yaml ./pnpm-workspace.yaml
-COPY ./turbo.json ./turbo.json
-COPY ./apps ./apps
+# Copy our packages
 COPY ./packages ./packages
 
-RUN pnpm install --frozen-lockfile
+# Install dependencies
+RUN bun install
+
+# Create a layer for build
+FROM dependencies AS build
+
+# Build the application
```

Comment:
As of right now this is purely for future intent

We have no js based build process

Elder God DHH got to us